api/lagoon/client/_lgraphql/organizations/organizationById.graphql

```diff
@@ -0,0 +1,13 @@
+query (
+  $id: Int!) {
+    organizationById(
+      id: $id) {
+      id
+      name
+      description
+      quotaProject
+      friendlyName
+      quotaGroup
+      quotaNotification
```

Comment:
add `quotaRoute`, and `quotaEnvironment` to this, and any other queries that retrieves the quotas